### PR TITLE
events/2020-online-get-together: Note about even later proposals

### PR DESCRIPTION
**Pull request by rkdarst**
- Don't make the deadline seem like a hard deadline, at this point
  unless someone is ready, they might give up.  Give some hope for
  later things.
- Review: is this a good idea, what we want to do?

### CODE REVIEW

File: events/2020-online-get-together.md [hunk 1]
```diff
@@ -59,8 +59,10 @@ Wednesday, 2 December:
 
 ### Proposal submissions
 
-[Abstract submission form](https://indico.neic.no/event/146/)
-is open. Submission deadline: 16th November 2020, 23:59 CET.
+[Abstract submission form](https://indico.neic.no/event/146/) is
+open. Submission deadline: 16th November 2020, 23:59 CET (but
+proposals will continue to be accept on a rolling, best-effort basis
+after this - please submit as soon as you can, even if not perfect).
```
Comment (wleoncio) [inline events/2020-online-get-together.md:65]: Sounds a little bit desperate IMO, maybe something like "we encourage you to submit..." sounds better.

Comment (rantahar) [reply events/2020-online-get-together.md]: I agree, and went ahead and changed the wording.

**Review by wleoncio — CHANGES_REQUESTED** [review]
Only thing left is to remove the duplicated "submit" and I think we're good to go!

**Review by wleoncio — APPROVED** [review]

**Comment by wleoncio** [comment]
Thanks @rantahar, I _just_ realized I could also have made the change online myself (too often I assume GH requires one to clone to local and push it back). 🤦🏽

**Comment by rantahar** [comment]
No problem. It was clearly useful that you reviewed the change I made :)

**Pull request merged**